### PR TITLE
Minify Theme Files And ignore Prettier

### PR DESCRIPTION
**Pull request by kunal202**
Fix #93

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "app/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "app/**/*.{js,jsx,json,graphql,gql}": [
     "prettier --write",
   ],
-  "api/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "api/**/*.{js,jsx,json,graphql,gql}": [
```
Comment (0xnmn) [inline .lintstagedrc]: You only have to ignore the theme css files not everything.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,10 @@
+gondor/app/styles/_custom.scss
+gondor/app/styles/_mixins.scss
+gondor/app/styles/_nunito.scss
+gondor/app/styles/_variables.scss
+gondor/app/styles/_gogo.style.scss
+gondor/app/styles/theme/styles.scss
+gondor/app/styles/theme/simple-line-icons/css/simple-line-icons.css
+gondor/app/styles/theme/iconsmind/style.scss
+gondor/app/styles/theme
+gondor/app/styles/_variables.scss
```
Comment (0xnmn) [inline .prettierignore]: Why have you added `gondor` at the start of each line? also adding `app/styles/theme` should work for all the nested files.

File: .prettierrc [hunk 3]
```diff
@@ -2,5 +2,5 @@
   "jsxBracketSameLine": true,
   "singleQuote": true,
   "trailingComma": "all",
-  "printWidth": 80,
+  "printWidth": 80
```
Comment (0xnmn) [inline .prettierrc]: remove this diff

**Review by 0xnmn — CHANGES_REQUESTED** [review]

File: .prettierrc (hunk 3 above)
Comment (kunal202) [reply .prettierrc]: Okay

File: .lintstagedrc (hunk 1 above)
Comment (kunal202) [reply .lintstagedrc]: Okay

File: .prettierignore (hunk 2 above)
Comment (kunal202) [reply .prettierignore]: Sorry, i was checking something and when it worked i forgot to remove this diff

File: .lintstagedrc [hunk 4]
```diff
@@ -1,8 +1,11 @@
 {
-  "app/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "app/**/*.{js,jsx,css,scss,json,graphql,gql}": [
     "prettier --write",
   ],
-  "api/**/*.{js,jsx,json,css,scss,graphql,gql}": [
+  "api/**/*.{js,jsx,css,scss,json,graphql,gql}": [
+    "prettier --write",
+  ],
+  "app/styles/**/!*.{css,scss}": [
```
Comment (0xnmn) [inline .lintstagedrc]: Remove this file from diff, I don't see any changes.

**Review by 0xnmn — CHANGES_REQUESTED** [review]

File: .lintstagedrc (hunk 4 above)
Comment (kunal202) [reply .lintstagedrc]: Done

**Comment by kunal202** [comment]
### Final preview after minification
<hr>

ezgif com-video-cutter

File: .prettierignore [hunk 5]
```diff
@@ -0,0 +1,7 @@
+app/styles/_custom.scss
```
Comment (0xnmn) [inline .prettierignore]: do not ignore this.

File: .lintstagedrc [hunk 6]
```diff
@@ -4,5 +4,5 @@
   ],
   "api/**/*.{js,jsx,json,css,scss,graphql,gql}": [
     "prettier --write",
-  ]
+  ],
```
Comment (0xnmn) [inline .lintstagedrc]: remove this from diff.

File: .prettierignore [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+app/styles/_mixins.scss
+app/styles/_nunito.scss
+app/styles/_variables.scss
+app/styles/_gogo.style.scss
+app/styles/_variables.scss
```
Comment (0xnmn) [inline .prettierignore:5]: do not ignore this.

File: app/styles/_variables.scss [hunk 8]
```diff
@@ -1,37 +1 @@
-$separator-color-light: #f3f3f3;
```
Comment (0xnmn) [inline app/styles/_variables.scss]: do not minify this file.

File: .lintstagedrc (hunk 6 above)
Comment (kunal202) [reply .lintstagedrc]: Done

File: .prettierignore (hunk 7 above)
Comment (kunal202) [reply .prettierignore:5]: Done

File: app/styles/_variables.scss (hunk 8 above)
Comment (kunal202) [reply app/styles/_variables.scss]: Done

**Comment by kunal202** [comment]
Changed according to the suggestions

File: .prettierignore [hunk 9]
```diff
@@ -0,0 +1,2 @@
+app/styles/_mixins.scss
```
Comment (0xnmn) [inline .prettierignore:1]: Only the custom.scss was not to be ignored

**Review by 0xnmn — CHANGES_REQUESTED** [review]

**Comment by kunal202** [comment]
is this good now ??

**Review by 0xnmn — APPROVED** [review]

**Pull request merged**